include/z64.h

```diff
@@ -1326,4 +1360,47 @@ typedef struct {
     /* 0x20 */ f32 unk_20;
 } UnkQuakeCalcStruct; // size = 0x24
 
+typedef struct {
+    /* 0x00 */ u32 idx;
+    /* 0x04 */ void* ptr;
+} UcodeInfo; // size = 0x8
```

Comment:
I believe UCode is used in essentially every other place besides those structs so we probably want to keep that consistency.
```suggestion
} UCodeInfo; // size = 0x8
```